CHANGELOG.md

```diff
@@ -196,6 +196,8 @@
 
 - Refactor `Tools` module with different Tools configuration processing based on ASAB-Config service changes (INDIGO Sprint 220107, [!222](https://github.com/TeskaLabs/asab-webui/pull/222))
 
+- Move all reducers to reducers folder (INDIGO Sprint 220204, [!232](https://github.com/TeskaLabs/asab-webui/pull/232))
```

Comment:
@titkov49 Technically its not all reducers, but only application reducers, since there are also reducers in particular modules of asab-ui

Reply:
Yeah, I meant all reducers which are loaded in Application.js directly. Others are part of modules and should stay on their places